src/components/Modal/SelectWalletModal/components/MetamaskStatusAlert.tsx

```diff
@@ -10,14 +10,23 @@ import {
 const MetamaskStatusAlert: FC<{
   metamaskNotInstalled?: boolean
   connectionRejected?: boolean
-}> = ({ metamaskNotInstalled, connectionRejected }) => {
+  isMetaMask: boolean
+}> = ({ metamaskNotInstalled, connectionRejected, isMetaMask }) => {
   const { active } = useWeb3React()
   if (metamaskNotInstalled) {
     return <MetamaskNotInstalledAlert />
   }
   if (connectionRejected) {
     return <WalletRejectedAlert />
   }
+  if (!isMetaMask) {
+    return (
+      <>
+        window.ethereum is not MetaMask. Make sure you have MetaMask extension
+        installed or MetaMask is your default wallet.
```

Comment:
I would chang it to:

```suggestion
        installed and MetaMask is your default wallet.
```